eclipse/che-plugin-registry
# chore(dependencies): Bump nodejs dependencies to avoid new CVE

### What does this PR do?
Update nodejs depencies to their latest versions

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?
Also bump ini component to avoid https://github.com/advisories/GHSA-qqgx-2p2h-9c37


### How to test this PR?
Check everything works (build/automation/pr-check) but most of it is used by CI script so checks should be green if everything works

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.

Change-Id: Ib3eb772d0e3b0b1224f1c9f7f4e71a629d405de1
Signed-off-by: Florent Benoit <fbenoit@redhat.com>
